test/logging/logging.go

```diff
@@ -115,22 +108,63 @@ func InitializeMetricExporter(context string) {
 		trace.UnregisterExporter(exporter)
 	}
 
-	logger := logger.Named(context)
+	l := logger.Named(context).Sugar()
 
-	exporter = &zapMetricExporter{logger: logger}
+	exporter = &zapMetricExporter{logger: l}
 	view.RegisterExporter(exporter)
 	trace.RegisterExporter(exporter)
 
 	view.SetReportingPeriod(metricViewReportingPeriod)
 	trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
 }
 
-// InitializeLogger initializes the base logger
-func InitializeLogger(logVerbose bool) {
-	logLevel := "info"
-	if logVerbose {
-		logLevel = "debug"
-	}
+func printFlags() {
+	flagList := make([]interface{}, 0)
+	flag.CommandLine.VisitAll(func(f *flag.Flag) {
+		flagList = append(flagList, f.Name, f.Value.String())
+	})
+	logger.Sugar().Debugw("Test Flags", flagList...)
```

Comment:
I think `Debugw` requires `zap.X()`.

Reply:
I'm afraid I don't understand your comment. [.Debugw()](https://godoc.org/go.uber.org/zap#SugaredLogger.Debugw) is a method of the sugared logger and the method does print out the test flags.

Reply:
We discussed; this wouldn't work for non-sugared logger, but does for sugared (which is used here).